src/tutorial/styles.scss

```diff
@@ -0,0 +1,36 @@
+@use '@carbon/styles/scss/spacing';
+@use '@carbon/styles/scss/type';
+@import '~@openmrs/esm-styleguide/src/vars';
+
+.description {
+  margin-bottom: spacing.$spacing-04;
+  @include type.type-style('body-compact-01');
+}
+
+.tutorialModal {
+  display: flex;
+  flex-direction: column;
+  gap: spacing.$spacing-04;
+
+  .tutorialItem {
+    border-bottom: 1px solid $ui-03;
+    padding-bottom: spacing.$spacing-02;
+
+    .tutorialTitle {
+      margin-bottom: spacing.$spacing-04;
+      @include type.type-style('heading-02');
+    }
+
+    .tutorialDescription {
+      color: $color-gray-70;
+      @include type.type-style('body-compact-01');
+    }
+
+    .walkthrough {
```

Comment:
This should work here, I guess:

@include type.type-style('body-compact-02');
color: $link-primary;

Reply:
is the `color: $link-primary` defined?
I'm getting an error as an Undefined variable.

Reply:
makes sense, even our primary color isn't blue.

Reply:
how about: @include type.type-style('body-compact-02');

Reply:
Yeah that works